pkg/utils/pdb/pdb.go

```diff
@@ -29,16 +29,16 @@ import (
 	podutil "sigs.k8s.io/karpenter/pkg/utils/pod"
 )
 
-// PDBLimits is used to evaluate if evicting a list of pods is possible.
-type PDBLimits struct {
+// Limits is used to evaluate if evicting a list of pods is possible.
+type Limits struct {
 	ctx        context.Context
 	clk        clock.Clock
 	kubeClient client.Client
```

Comment:
I have a general question about why we need to store the kubeClient, context, and clock in this data structure when it doesn't appear to be a global construct but more of a data holder? Are there places where we are utilizing this client ad-hoc?

Reply:
Good call. It really only needs to be a list of PDBItems. Changed it to be a type of list of pdbs.